### ISSUE TITLE
Images selected

### ISSUE THREAD

**Issue by G-200010** [h751410234/RemoteSensingTeacher#2]
In xView, is your strategy on choosing images by regarding 11:Fixed-wing Aircraft, 12:Small Aircraft, 13:Passenger/Cargo Plane and15:Helicopter as plane, 40:Maritime Vessel, 41:Motorboat, 42:Sailboat, 44:Tugboat, 45:Barge, 47:Fishing Vessel, 49:Ferry, 50:Yacht and 51:Container Ship as ship, 86:Storage Tank as storage tank and get a total of 216 images？ 
Hope for your reply.
<img width="762" alt="5555555" src="https://github.com/h751410234/RemoteSensingTeacher/assets/172945648/baf2a8c9-0e7f-455c-9058-d17d95e66e03">

**Comment by h751410234** [comment]
1.My experimental setup:
DOTA: The official DOTA1.0 dataset is used, with training and test sets already divided by the official organization.
In experiments, the training set is the official one, and the test set is the official validation set 
xView: Only the official version of the training set is used (as it only participates in training as the source domain).
﻿
2.Data Preprocessing
1）For both DOTA and xView datasets, the image sizes vary and are generally large. Therefore, sliding window slicing is necessary (a common operation in remote sensing image processing, to make each image a fixed size). In the experiments section, I have mentioned that the dataset is supposed to be 800x800 in size, with an overlap of 100 pixels.
2）Image selection, since the experiments include only three categories: ships, airplanes, and oil tubes, images that do not contain these three categories are not used (and can be deleted).
3）Due to the fine categorization in xView, for instance, taking ships as an example: the xView dataset has detailed annotations (Sailboat, Barge, etc.), which are inconsistent with DOTA’s broader categorization (only having 'ship' as a category). Therefore, all categories related to 'ship' in xView are merged to align with DOTA's annotations.
﻿
Specific merging list for xView categories is as follows:
plane: 'Fixed-wing Aircraft', 'Small Aircraft', 'Cargo Plane'；
ship: 'Maritime Vessel', 'Motorboat', 'Sailboat', 'Tugboat', 'Barge', 'Fishing Vessel', 'Ferry', 'Yacht', 'Container Ship', 'Oil Tanker'.
﻿
3.Number of images
792 DOTA image and 216 xView image , these should be introduced as part of the dataset, regardless of the specific number of images used in the experiments.
The number of images after preprocessing is as follows:
DOTA: Number of image slices (800x800), training set: 4805 images, validation set: 1577 images.
xView: Number of image slices (800x800), training set: 1878 images.
﻿

**Comment by G-200010** [comment]
Really appreciate for your reply. But I think I do not fully understand this statement : '792 DOTA image and 216 xView image , these should be introduced as part of the dataset, regardless of the specific number of images used in the experiments.'  Is that mean the number 216 and 792 is just a description of the datasets, but not really be concerned about the dataset choosing strategy and the specific implementation of the experiment？

**Comment by G-200010** [comment]
The number of images after preprocessing is as follows:
DOTA: Number of image slices (800x800), training set: 4805 images, validation set: 1577 images.
xView: Number of image slices (800x800), training set: 1878 images.
This process should be done on my own or has been achieved in the code?

**Comment by h751410234** [comment]
(1) The xView to DOTA experiment was not proposed in my paper. The 792 DOTA images and 216 xView images are sourced from the paper "FADA: Feature Aligned Domain Adaptive Object Detection in Remote Sensing Imagery." You might be able to get the information you need from that paper.
(2) The data preprocessing methods mentioned above need to be implemented offline by yourself.

**Comment by G-200010** [comment]
Really appreciate for your reply. Thanks a lot.
